File: internal/sast/fixtures/python/mass_negative.py
# internal/sast/fixtures/python/mass_negative.py
from django.http import JsonResponse
from .models import User

ALLOWED_FIELDS = {"name", "email"}

def create_user(request):
    payload = {k: request.POST[k] for k in ALLOWED_FIELDS if k in request.POST}
    user = User.objects.create(**payload)
    return JsonResponse({"id": user.id})

File: internal/sast/fixtures/javascript/mass_negative.js
// internal/sast/fixtures/javascript/mass_negative.js
const express = require("express");
const User = require("./models/user");
const app = express();
app.use(express.json());

app.post("/users", async (req, res) => {
  const { name, email } = req.body;
  const user = await User.create({ name: String(name), email: String(email) });
  res.json(user);
});
